ISPN-9416-FIX Updating multimap doc for supporting duplicates (#10407)
infinispan/infinispan: documentation/src/main/asciidoc/topics/multimapcache.adoc

In almost every case the Multimap Cache will behave as a regular Cache, but some limitations exist in the current version, as follows:

=== Support for duplicates

> Duplicates can optionally be supported. This means that the multimap can contain duplicate key-value pairs.

Reviewer: ```suggestion
A multimap can be configured to store duplicate values for a single key. A duplicate is determined by the value's `equals` method.
```

> Whenever put method is called, if the key-value pair already exist and multimap is configured to support duplicates, this key-value pair will be added.

Reviewer: ```suggestion
Whenever the put method is called, if multimap is configured to support duplicates, the key-value pair will be added to the collection.
```

> In the case where duplicates are supported, all the key-value pairs will be removed.

Reviewer: ```suggestion
Invoking remove on the multimap will remove all duplicates if present.
```
